### ISSUE TITLE
Can not start reaction by cli, freezed at setting up plugin imports

### ISSUE THREAD

**Issue by nie-xin** [reactioncommerce/reaction#1491]
The `reaction init` is succeed, but when I run `reaction`, it freezes at 'Setting up plugin imports...' and never go forward after a longtime.

```
$ reaction run

Using settings file at settings/dev.settings.json

Setting up plugin imports...
```

```
$ reaction -v

Node: 6.2.2
NPM: 3.9.5
Reaction: 0.16.1
Reaction CLI: 0.4.21
```

**Comment by jshimko** [comment]
What OS are you on?  Did you create the app with `reaction init`?  Have you edited any code?

**Comment by nie-xin** [comment]
@jshimko I'm on mac os, and yes it is created successfully with `reaction init` and no code modified.

**Comment by jshimko** [comment]
Well, it's a unique problem that nobody else appears to be having, so can you try running `reaction init` again and starting with a fresh copy?  Also, how long did you wait?  It will definitely take at least 30 seconds to build the app the first time.

**Comment by nie-xin** [comment]
Yes its really wired! I waited about 10 mins and it did't progress. I will try to create another project when I back home, and give you more feedback then. Maybe has sth to do with my laptop setting.

**Comment by jshimko** [comment]
Ok.  Yeah, it takes a little while, but definitely not 10 mins.  Something else is wrong then.

Usually the next thing you would see is MongoDB starting up, so perhaps that's getting hung up.  I hate to say it, but sometimes the classic computer reboot solves issues like this too.  So I'd try that as well.

**Comment by nie-xin** [comment]
@jshimko thanks a lot. Will try again 👍

**Comment by willboudle** [comment]
@nie-xin , curious if you found the solution, I'm running into the same thing, Windows 10, fresh install, also tried with a 4.1 version of Node.

`reaction -v

Node: 6.9.1
NPM: 3.10.9
Reaction: 0.17.1
Reaction CLI: 0.5.1`

**Comment by hrath2015** [comment]
@willboudle  try below:
Modify  .meteor\packages file line 71
from ``momentjs:moment``
to ``momentjs:moment@=2.15.1``

**Comment by jshimko** [comment]
@hrath2015 Have you tried 2.16.0?

**Comment by hrath2015** [comment]
Nope.  I will try and let u know.

**Comment by hrath2015** [comment]
@jshimko  updating to  meteor 1.4.2.1 solves it.  I did a build on branch 0.18.0.

**Comment by hrath2015** [comment]
@willboudle  latest pull from Dev branch should address this issue. It was merged as part of PR  #1575.

**Comment by willboudle** [comment]
thank you, got it working.
